sdk/storage/azblob/README.md

```diff
@@ -0,0 +1,224 @@
+client's `NewServiceClientFromConnectionString` method. The connection string can be found in your storage account in
+the [Azure Portal][azure_portal_account_url] under the "Access Keys" section or with the following Azure CLI command:
+
+```bash
+az storage account show-connection-string -g MyResourceGroup -n MyStorageAccount
+```
+
+```golang
+connStr := "DefaultEndpointsProtocol=https;AccountName=<myAccountName>;AccountKey=<myAccountKey>;EndpointSuffix=core.windows.net"
+serviceClient, err := azblob.NewServiceClientFromConnectionString(connStr, nil)
+```
+
+##### Creating the client from a SAS token
+
+To use a [shared access signature (SAS) token][azure_sas_token], provide the token as a string. If your account URL
+includes the SAS token, omit the credential parameter. You can generate a SAS token from the Azure Portal
+under [Shared access signature](https://docs.microsoft.com/rest/api/storageservices/create-service-sas) or use
+the `ServiceClient.GetAccountSASToken` or `ContainerClient.GetContainerSASToken()` methods.
+
+```golang
+cred, err := azblob.NewSharedKeyCredential("myAccountName", "myAccountKey")
+handle(err)
+service, err := azblob.NewServiceClient("https://<myAccountName>.blob.core.windows.net", cred, nil)
+
+resources := azblob.AccountSASResourceTypes{Service: true}
+permission := azblob.AccountSASPermissions{Read: true}
+start := time.Now()
+expiry := start.AddDate(1, 0, 0)
+sasUrl, err := service.GetAccountSASToken(resources, permission, start, expiry)
+handle(err)
+
+sasService, err := azblob.NewServiceClient(sasUrl, azcore.AnonymousCredential(), nil)
+handle(err)
+```
+
+- For more detailed examples, please refer to zt_examples_test.go.
+
+### Clients
+
+Three different clients are provided to interact with the various components of the Blob Service:
+
+1. **`ServiceClient`**
+    * Get and set account settings.
+    * Query, create, and delete containers within the account.
+
+2. **`ContainerClient`**
+    * Get and set container access settings, properties, and metadata.
+    * Create, delete, and query blobs within the container.
+    * `ContainerLeaseClient` to support container lease management.
+
+3. **`BlobClient`**
+    * `AppendBlobClient`, `BlockBlobClient`, and `PageBlobClient`
+    * Get and set blob properties.
+    * Perform CRUD operations a given blob.
+    * `BlobLeaseClient` to support blob lease management.
+
+## Troubleshooting
+
+### Error Handling
+
+All I/O operations will return an `error` that can be investigated to discover more information about the error. In
+addition, you can investigate the raw response of any response object:
+
+```golang
+resp, err := serviceClient.CreateContainer(context.Background(), "testcontainername", nil)
+if err != nil {
+    err = errors.As(err, azcore.HTTPResponse)
+    // handle err ...
+}
+```
+
+### Logging
+
+This module uses the classification based logging implementation in azcore. To turn on logging
+set `AZURE_SDK_GO_LOGGING` to `all`.
```

Comment:
@jhendrixMSFT is this environment variable still valid?

Reply:
Yes see [here](https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/internal/log/log.go#L105).  The recent PR to `azcore` also updated the docs with this.